src/x509.c

```diff
@@ -7523,6 +7523,45 @@ int wolfSSL_i2d_X509(WOLFSSL_X509* x509, unsigned char** out)
     return derSz;
 }
 
+#ifdef WOLFSSL_DUAL_ALG_CERTS
+int GeneratePreTBS(DecodedCert* cert, byte *der, int derSz) {
```

Comment:
This is marked as a PUBLIC API.  If you do need this public it should have leading `wc_`.

Reply:
Fixed.